projects/plugins/boost/app/modules/optimizations/minify/class-css.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Automattic\Jetpack_Boost\Modules\Optimizations\Minify;
+
+use Automattic\Jetpack_Boost\Contracts\Pluggable;
+
+class Css extends Minify implements Pluggable {
```

Comment:
During review, I found it difficult to search for `new Css` or `Css` - many things are called `Css`, so perhaps a more specific name would be better for this class?

Reply:
Fair point. I did a silly thing where I declared them without `Minify` but aliased them wherever they were used with the prefix. Updated with 286067045c360986184d090cb638bc9956fd7bcf. Specifically:

https://github.com/Automattic/jetpack/blob/286067045c360986184d090cb638bc9956fd7bcf/projects/plugins/boost/app/modules/optimizations/minify/class-minify-js.php#L7
https://github.com/Automattic/jetpack/blob/286067045c360986184d090cb638bc9956fd7bcf/projects/plugins/boost/app/modules/optimizations/minify/class-minify-css.php#L7